app/src/main/java/ar/scacchipa/twittercloneapp/fragment/FragmentAuthWebDialog.kt

```diff
@@ -33,10 +34,13 @@ class FragmentAuthWebDialog : Fragment() {
             when (token) {
 
                 is ResponseDomain.Success<*> -> {
```

Comment:
Github does not let me write on not related to this specific pr, but I think we could improve the name of the viewmodel´s livedata. responseDomain is not much descreptive.